### ISSUE TITLE
En qu'administrateur je peux faire des tableaux

### ISSUE THREAD

**Issue by Olivier-Marcellin** [demarche-numerique/demarche.numerique.gouv.fr#7375]
Demande exprimée par les équipes du Ministère de la Culture

Priorisation urgente côté ministère de la Culture

Et priorisation faible pour l'optimisation de la présentation du type de champ "Date et heure " (sur la même ligne : champ "Date " + listes déroulantes "Heure " et "Minutes ")

Constat 

Une augmentation de la longueur du formulaire, en raison d’une présentation systématique des champs les uns en-dessous des autres.

Verbatim admin. : "assez régulièrement on arrive à se passer des tableaux en mettant un bloc répétable à la place du tableau, mais quand c’est des suites de nombres, par exemple dans le formulaire CERFA demande de subvention, on demande les ressources, les moyens humains affectés au projets en nombre de personnes et en équivalent temps plein de travail. Dans le CERFA c’est un tableau de 5 lignes, dans notre formulaire c’est une étape extrêmement longue puisqu’on est obligé de tout dupliquer plusieurs fois, donc ça c’est très très long, c’est un exemple"

Amélioration

Possibilité de factoriser les informations et de présenter certains champs sur la même ligne sous forme de tableau / grille.

US : En tant qu'administrateur je peux faire des tableaux afin de simplifier les formulaires

Voir les illustrations ci-dessous

**Comment by tchak** [comment]
c'est pas très cher – c'est "juste" un autre rendu d'un bloc répétable

**Comment by tchak** [comment]
c'est pas très cher – c'est "juste" un autre rendu d'un bloc répétable

**Comment by mfo** [comment]
> c'est pas très cher – c'est "juste" un autre rendu d'un bloc répétable

juste une petite craintes avec certains types de champs (textarea, map dans un tableau...). donc faudrait une liste des champs compatible et la je serai rassuré.

**Comment by tchak** [comment]
@mfo oui, absolument. On exclut déjà certains types de champ des répétitions. Ici la liste d'exclusion sera probablement plus étendue, mais la logique existe déjà.
